packages/terra-core-docs/src/terra-dev-site/test/content-container/ScrollableContentContainer.test.jsx

```diff
@@ -12,18 +12,18 @@ const display1 = (
     verification of personal information, updating medical history, and making necessary changes to insurance or contact details. Hospitals may
     request patients to review and confirm their information during subsequent visits or at designated intervals. Accurate and comprehensive patient
     details play a vital role in providing effective healthcare. They serve as a foundation for medical assessments, treatment planning, and continuity
-    of care, enabling healthcare providers to deliver personalized and efficient services to patients.Begin filling out the form by entering your personal
+    of care, enabling healthcare providers to deliver personalised and efficient services to patients.Begin filling out the form by entering your personal
```

Comment:
Personalized is correct; that is the US spelling. Since the documentation does not have internationalization unfortunately, we should be consistent with using US English. (e.g. we use `color` instead of `colour`)